src/ReactiveUI.Winforms/ActivationForViewFetcher.cs

```diff
@@ -13,23 +13,25 @@ namespace ReactiveUI.Winforms
 {
     public class ActivationForViewFetcher : IActivationForViewFetcher, IEnableLogger
     {
+        private Lazy<bool> _isDesignModeCache = null;
```

Comment:
You don't really need the = null; on the end of that. It'll default to that regardless.